pyuntl/untl_structure.py

```diff
@@ -457,6 +458,42 @@ def generate_form_data(self, **kwargs):
         # Create the form object.
         return FormGenerator(**kwargs)
 
+    def make_hidden(self):
+        """Make an unhidden record hidden."""
+        for element in self.children:
+            if element.tag == 'meta' and element.qualifier == 'hidden':
+                # Make the element hidden.
+                if element.content == 'False':
+                    element.content = 'True'
+                    return None
```

Comment:
I think this `return None` should be outdented one level assuming we want to return after finding meta tag with hidden qualifier no matter what the content value is.